src/main/resources/localization/messages-pl.properties

```diff
@@ -0,0 +1,78 @@
+error.permissions.user = Aby użyć tej komendy potrzbujesz permisji Manage Server. 
+error.permissions.perms_or_role = Aby użyć tej komendy potrzebujesz permisji Manage Server albo roli {0}. 
+error.permissions.perms_no_role = Aby użyć tej komendy potrzbujesz permisji Manage Server. Rola zarządcy giveawayi nie została jeszcze ustawiona. Administrator może ją ustawić za pomocą `{0}`
+error.permissions.bot = Na tym kanale brakuje mi tych permisji: `Read Messages`, `Send Messages`, `Read Message History`, `Embed Links`. Napraw te permisje albo zaproś mnie jeszcze raz [tutaj]({0}).
+error.invalid_time.format = Nie mogłem zamienić `{0}` na poprawną ilość czasu! 
+error.invalid_time.min = Podany czas ({0}) jest krótszy niż czas minimalny ({1})!
+error.invalid_time.max = Podant czas ({0}) jest dłuższy niż czas maksymalny ({1})!
+error.max_giveaways.guild = Istnieją już {0} aktywne giveawaye an tym serwerze!  
+error.max_giveaways.channel = Istnieją już {0} aktywne giveawaye an tym kanale!
+error.invalid_winners.format = Nie mogłem zamienić `{0}` na poprawną ilość wygranych!
+error.invalid_winners.max = Podana ilość wygranych ({0}) nie mieście się między 1, a {1}!
+error.invalid_prize.length = Długość nagrody nie może być dłuższa niż {0} znaków! 
```

Comment:
Correct the word "nagrody" to "cena"

Reply:
That would depend on the context it is used in, seeing as how it's about a giveaway prize I think the word "nagroda" is more appropriate here